src/App.tsx

```
@@ -1,6 +1,18 @@
 import React from "react";
 
 export const App: React.FC = () => {
-	return <div style={{ padding: "2rem" }}>Hello</div>;
+	const fruits = ["Apple", "Banana", "Grapes", "Mango"];
+
+	return (
+		<div style={{ padding: "2rem" }}>
+			<h1>Fruit List</h1>
+			<ul>
+				{fruits.map((fruit) => (
+					<li>{fruit}</li>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)